dockerize.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"

```

Comment:
Space after shebang

Reply:
oops I forgot to address this when you first mentioned it. Fixed!